### PR TITLE
 docs/kubernetes *: Update the description about access radondb mysql #340

### LINKED ISSUE #340: [documentation] Simplify way to connect Mysql.
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

It doesn't need to distinguish whether it is connected to MySQL in the same ns, all use `<service name>. <namespace>` to connect.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by molliezhang**
### What type of PR is this?

/documentation
/enhancement

### Which issue(s) this PR fixes?

Fixes #340 

### What this PR does?

Summary:

### Special notes for your reviewer?
@acekingke @andyli029 @runkecheng @qianfen2021 @KID-G

### CODE REVIEW

File: docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md [hunk 1]
```diff
@@ -157,62 +155,55 @@ demo-radondb-mysql   3/3     25h
...
+> 
+> 准备可用于连接 MySQL 的客户端。
+> 
+> 使用外网客户端连接可能会出现 `SSL connection error`，需要加上 `--ssl-mode=DISABLE` 参数，关闭 SSL。
 
-### 客户端与 RadonDB MySQL 在同一 NameSpace 中
+- 当客户端的与数据库部署在不同 Kubernetes 集群，请参考 [Kubernetes 访问集群中的应用程序](https://kubernetes.io/zh/docs/tasks/access-application-cluster/)，配置端口转发、负载均衡等连接方式。
 
-当客户端与 RadonDB MySQL 集群在同一个 NameSpace 中时，可使用 leader/follower service 名称代替具体的 ip 和端口。
+- 在 Kubernetes 集群内，支持选择使用 `service_name` 或者 `service_IP` 方式访问 RadonDB MySQL。
```
Comment (runkecheng) [inline docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md:168]: service_IP -> ClusterIP，see [k8s service](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)

File: docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md [hunk 2]
```diff
@@ -157,62 +155,55 @@ demo-radondb-mysql   3/3     25h
...
 
-- 连接从节点(只读节点)。
-  ```bash
-  mysql -h <follower service 名称> -u <用户名> -p
-  ```
+* 连接主节点(读写节点)
 
-### 客户端与 RadonDB MySQL 不在同一 NameSpace 中
+    ```shell
+    $ mysql -h <leader_service_name> -u <user_name> -p
```
Comment (runkecheng) [inline docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md:175]: Add ns like `sample-leader.default`

**Review by runkecheng — CHANGES_REQUESTED** [review]

File: docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md (hunk 2 above)
Comment (molliezhang) [reply docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md:175]: ok

File: docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md (hunk 1 above)
Comment (molliezhang) [reply docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md:168]: ok

File: docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md [hunk 3]
```diff
@@ -157,62 +155,55 @@ demo-radondb-mysql   3/3     25h
...
 
-您需要准备一个用于连接 RadonDB MySQL 的客户端。
+> **注意**
+> 
+> 准备可用于连接 MySQL 的客户端。
+> 
+> 使用外网客户端连接可能会出现 `SSL connection error`，需要加上 `--ssl-mode=DISABLE` 参数，关闭 SSL。
 
-### 客户端与 RadonDB MySQL 在同一 NameSpace 中
+- 当客户端的与数据库部署在不同 Kubernetes 集群，请参考 [Kubernetes 访问集群中的应用程序](https://kubernetes.io/zh/docs/tasks/access-application-cluster/)，配置端口转发、负载均衡等连接方式。
```
[the remark this answers is not in the document]
Comment (molliezhang) [reply docs/kubernetes/deploy_radondb-mysql_on_kubernetes.md:166]: ok

**Review by runkecheng — APPROVED** [review]
lgtm

**Comment by andyli029** [comment]
LGTM.

**Pull request merged**